testing/inputs/implementations/pending/ercim.html

```diff
@@ -0,0 +1,46 @@
+<div class="impl" id="impl-ercim">
```

Comment:
Please remove for now and confirm with @draggett if the description is up to date.
It can then be put back in a separate PR.